gsplat/rasterize.py

```diff
@@ -139,14 +144,15 @@ def backward(ctx, v_out_img):
             img_width,
             gaussian_ids_sorted.contiguous(),
             tile_bins,
-            xys.contiguous(),
-            conics.contiguous(),
-            colors.contiguous(),
-            opacity.contiguous(),
-            background.contiguous(),
-            final_Ts.contiguous(),
-            final_idx.contiguous(),
-            v_out_img.contiguous(),
+            xys.contiguous().cuda(),
+            conics.contiguous().cuda(),
+            colors.contiguous().cuda(),
+            opacity.contiguous().cuda(),
+            background.contiguous().cuda(),
+            final_Ts.contiguous().cuda(),
+            final_idx.contiguous().cuda(),
+            v_out_img.contiguous().cuda(),
```

Comment:
I think ruilong removed the cuda() for these, could you take them out too?

Reply:
fixed but should .cuda()s still be in NDRasterize backwards?